### PR TITLE
Update to snake case from kebab case in values.yaml for sync-integration-tests

### PR DESCRIPTION
**Pull request by championshuttler**
Fixes part of  #258

### CODE REVIEW

File: testing/sync_integration_tests/BUILD.bazel [hunk 1]
```diff
@@ -7,7 +7,7 @@ kubectl_patch(
     name = "sync_integration_tests",
     namespace = project.namespace,
     resource_type = "qjob",
-    resource_name = "kubecf-sync-integration-tests",
+    resource_name = "kubecf-sync_integration_tests",
```
Comment (f0rmiga) [inline testing/sync_integration_tests/BUILD.bazel]: This is wrong. This is the k8s resource name, and k8s doesn't allow `_`. In fact, cf-operator converts any `_` to `-`.

Comment (f0rmiga) [inline testing/sync_integration_tests/BUILD.bazel]:
```suggestion
    resource_name = "kubecf-sync-integration-tests",
```

Comment (fargozhu) [reply testing/sync_integration_tests/BUILD.bazel]: @f0rmiga what about the "-" after the "kubecf" word?

**Comment by viovanov** [comment]
Not something we want to pursue right now.
Any changes to the values keys should come with deprecation.

**Pull request closed without merging**